sonar-plugin-api/src/main/java/org/sonar/api/server/ws/Response.java

```diff
@@ -41,12 +41,17 @@
      */
     Stream setStatus(int httpStatus);
 
+    /**
+     * Response stream. Beware that proper error management is not possible.
+     */
     OutputStream output();
   }
 
   /**
-   * Non streamable {@link JsonWriter}. Response is written when resource is closed.
+   * Streamable {@link JsonWriter}. Beware that proper error management is not possible.
+   * @deprecated since 7.0. Please use {@link Stream#output()} explicitly.
    */
+  @Deprecated
   JsonWriter newJsonWriter();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fteryk%2Fws-no-stream&issues=AWC8zLn6_Q-0dz4JA0yU&open=AWC8zLn6_Q-0dz4JA0yU)

[ISSUE_KEY]: AWC8zLn6_Q-0dz4JA0yU